bind/gencffi.go

```diff
@@ -87,8 +88,8 @@ class _cffi_helper(object):
 
     @staticmethod
     def cffi_cgopy_cnv_py2c_string(o):
-        if _PY3:
-            o = o.encode('ascii')
+        if _PY3 and isinstance(o, str) or not _PY3 and isinstance(o, unicode):
```

Comment:
could you put explicit parentheses?
with the mix of `and` and `or`, it's a bit hard to understand exactly what this is doing (well, of course "everybody" knows the precedence of `or` and `and`, but...)

Reply:
Good idea -- it also avoids having to think about the weird operator chaning semantics in Python.